docs/docs/user-setup.md

```diff
@@ -193,14 +193,31 @@ To flash the firmware, first put your board into bootloader mode by double click
 or the one that is part of your keyboard). The controller should appear in your OS as a new USB storage device.
 
 Once this happens, copy the correct UF2 file (e.g. left or right if working on a split), and paste it onto the root of that USB mass
-storage device. Once the flash is complete, the controller should automatically restart, and load your newly flashed firmware.
+storage device. Once the flash is complete, the controller should automatically restart, and load your newly flashed firmware. It is
+recommended that you test your keyboard works over USB first to rule out hardware issues, before trying to connect to it wirelessly.
+
+:::caution Split keyboards
+
+For split keyboards, only the central half (typically the left side) will send keyboard outputs over USB or advertise to other devices
+over bluetooth. Peripheral half will only send keystrokes to the central once they are paired and connected. For this reason it is
+recommended to test the left half of a split keyboard first.
+
+:::
 
 ## Wirelessly Connecting Your Keyboard
 
-ZMK will automatically advertise itself as connectable if it is not currently connected to a device. You should be able to see your keyboard from the bluetooth scanning view of your laptop or phone / tablet. It is reported by some users that the connections with Android / iOS devices are generally smoother than with laptops, so if you have trouble connecting, you could try to connect from your phone or tablet first to eliminate any potential hardware issues.
+ZMK will automatically advertise itself as connectable if it is not currently connected to a device. You should be able to see your keyboard from the bluetooth scanning view of your computer or phone / tablet. It is reported by some users that the connections with Android / iOS devices are generally smoother than with laptops, so if you have trouble connecting, you could try to connect from your phone or tablet first to eliminate any potential hardware issues with bluetooth receivers.
```

Comment:
Really detailed stuff! Perhaps "your computer or phone / tablet" can be shortened to just "your device"?

Reply:
We could probably replace it here but this is the most beginner-friendly page of the docs, so maybe it doesn't hurt to be a bit specific/give examples.

In general I am scared people might think "device" refers to the keyboard, though probably not in this specific spot :)